packages/connection-provider/src/main/ts/interfaces.ts

```diff
@@ -17,15 +16,6 @@ export type IConsulDiscovery = {
   serviceName: string
 }
 
-export type IConnectionParams = {
```

Comment:
1. Types refactoring will bring a breaking change
2. Consul module _implements_ conn provider contract, so the types should stay here or be moved upper to the common module.